### PR TITLE
Documentations folder

### PR DESCRIPTION
**Pull request by carsonclarke570**
Made folder filled with Markdown files to make notes about design choices and idea.

### CODE REVIEW

File: docs/GAME.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+A general purpose object identified by a unique id. Every entity type should have its own set of unique identifiers.
+
+#### Component
+
+Raw data to describe an aspect of the object. Typically implemented using a struct or class.
+
+#### System
+
+Each system runs continuously and preforms global actions on every entity that possesses a Component of the same type.
```
Comment (marthinwurer) [inline docs/GAME.md]: performs
```suggestion
Each system runs continuously and performs global actions on every entity that possesses a Component of the same type.
```

File: docs/GAME.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+### Combat `DEV`
+
+I'm feeling ARPG zelda style but with more combat options.
+
+### Stats
+
+Each statistic should have equal beneficial impact on gameplay. There should not be a statistic that does not directly impact play style. My suggestion for stats:
+
+`Constitution` - Defines the character's health. Tanky builds should have high CON.
+`Defense` - Defines how much damage a character takes. Armored builds should habe high DEF.
```
Comment (marthinwurer) [inline docs/GAME.md:36]: How does this differ from Constitution? Is this like armour?

**Review by marthinwurer — CHANGES_REQUESTED** [review]
Mostly looks good!

File: docs/GAME.md (hunk 2 above)
Comment (carsonclarke570) [reply docs/GAME.md:36]: A class with high defense will take less damage, while a class with high constitution will have more HP. It's possible that both these qualities would better serve being treated as one attribute but usually games have them seperate.

**Review by marthinwurer — APPROVED** [review]

**Pull request merged**